dashboard/app/assets/stylesheets/application.css.scss

```diff
@@ -370,10 +370,12 @@ $small-footer-standard-width: 400px;
   z-index: 1;
```

Comment:
It doesnt currently overlap with anything, but it's probably good to bump this too for consistency